variables.tf

```diff
@@ -469,3 +469,47 @@ variable "graphdb_enable_userdata_scripts_on_reboot" {
   type        = bool
   default     = false
 }
+
+# KMS CMK
+
+variable "enable_cmk" {
+  description = "Enable CMK for encryption. If false, use AWS managed key."
+  type        = bool
+  default     = false
+}
+
+variable "kms_master_key_id" {
+  description = "ARN of the Customer Managed Key (CMK)."
+  type        = string
+  default     = ""
+}
+
+variable "sns_cmk_policy" {
+  description = "The IAM policy JSON document."
+  type        = string
+  default     = ""
+}
+
+variable "tags" {
```

Comment:
This is not used anywhere (because it's not passed in the main.tf). Why do you need this? We already have `common_tags` that are used in provider.tf, effectively adding them to every resource automatically.